### ISSUE TITLE
Unable to update netcdf package

### ISSUE THREAD

**Issue by gongled** [essentialkaos/kaos-repo#173]
Hello, @andyone.

It looks like `gdal` package is built with earlier version of `netcdf` than EK has in the repository. 

```
Resolving Dependencies
--> Running transaction check
---> Package netcdf.x86_64 0:4.4.1.1-0.el6 will be updated
--> Processing Dependency: libnetcdf.so.11()(64bit) for package: gdal-1.11.5-0.el6.x86_64
---> Package netcdf.x86_64 0:4.5.0-0.el6 will be an update
--> Finished Dependency Resolution
Error: Package: gdal-1.11.5-0.el6.x86_64 (@kaos-release)
           Requires: libnetcdf.so.11()(64bit)
           Removing: netcdf-4.4.1.1-0.el6.x86_64 (@kaos-release)
               libnetcdf.so.11()(64bit)
           Updated By: netcdf-4.5.0-0.el6.x86_64 (kaos-release)
               Not found
 You could try using --skip-broken to work around the problem
```

**Comment by andyone** [comment]
Hi! I will try to rebuild this package with the latest version of `netcdf`.

**Comment by andyone** [comment]
Fix released as part of #171
